Index: lede/target/linux/ar71xx/image/legacy-devices.mk
===================================================================
--- lede.orig/target/linux/ar71xx/image/legacy-devices.mk
+++ lede/target/linux/ar71xx/image/legacy-devices.mk
@@ -213,7 +213,7 @@ LEGACY_DEVICES += YUN_16M
 define LegacyDevice/F9K1115V2
   DEVICE_TITLE := Belkin AC1750DB (F9K1115V2)
   DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport \
-	kmod-ath10k ath10k-firmware-qca988x
+	kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += F9K1115V2
 
@@ -295,13 +295,13 @@ LEGACY_DEVICES += DIR505A1
 
 define LegacyDevice/DGL5500A1
   DEVICE_TITLE := D-Link DGL-5500 rev. A1
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += DGL5500A1
 
 define LegacyDevice/TEW823DRU
   DEVICE_TITLE := TRENDNet TEW-823DRU
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += TEW823DRU
 
@@ -370,7 +370,7 @@ LEGACY_DEVICES += dLAN_pro_500_wp
 
 define LegacyDevice/dLAN_pro_1200_ac
   DEVICE_TITLE := devolo dLAN pro 1200+ WiFi ac
-  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += dLAN_pro_1200_ac
 
@@ -382,13 +382,13 @@ LEGACY_DEVICES += ESR900
 
 define LegacyDevice/ESR1750
   DEVICE_TITLE := EnGenius ESR1750
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += ESR1750
 
 define LegacyDevice/EPG5000
   DEVICE_TITLE := EnGenius EPG5000
-  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += EPG5000
 
Index: lede/target/linux/ar71xx/image/nand.mk
===================================================================
--- lede.orig/target/linux/ar71xx/image/nand.mk
+++ lede/target/linux/ar71xx/image/nand.mk
@@ -89,6 +89,6 @@ LEGACY_DEVICES += WNDR4300V1
 
 define LegacyDevice/NBG6716
   DEVICE_TITLE := Zyxel NBG 6716
-  DEVICE_PACKAGES := kmod-rtc-pcf8563 kmod-ath10k ath10k-firmware-qca988x
+  DEVICE_PACKAGES := kmod-rtc-pcf8563 kmod-ath10k ath10k-firmware-qca988x-ct
 endef
 LEGACY_DEVICES += NBG6716
