--- a/feeds/luci/applications/luci-app-olsr-services/Makefile
+++ b/feeds/luci/applications/luci-app-olsr-services/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=Show services announced with the nameservice plugin
-LUCI_DEPENDS:=+luci-app-olsr +olsrd +olsrd-mod-nameservice
+LUCI_DEPENDS:=+luci-app-olsr +olsrd-stable-git +olsrd-stable-git-mod-nameservice
 
 include ../../luci.mk
 
--- a/feeds/luci/applications/luci-app-olsr-viz/Makefile
+++ b/feeds/luci/applications/luci-app-olsr-viz/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=OLSR Visualisation
-LUCI_DEPENDS:=+luci-app-olsr +olsrd +olsrd-mod-txtinfo
+LUCI_DEPENDS:=+luci-app-olsr +olsrd-stable-git +olsrd-stable-git-mod-txtinfo
 
 include ../../luci.mk
 
--- a/feeds/luci/applications/luci-app-olsr/Makefile
+++ b/feeds/luci/applications/luci-app-olsr/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=OLSR configuration and status module
-LUCI_DEPENDS:=+olsrd +olsrd-mod-jsoninfo +luci-lib-luaneightbl +luci-lib-json
+LUCI_DEPENDS:=+olsrd-stable-git +olsrd-stable-git-mod-jsoninfo +luci-lib-luaneightbl +luci-lib-json
 
 include ../../luci.mk
 
